prow/plank/controller.go

```diff
@@ -112,24 +158,31 @@ func (c *Controller) Sync() error {
 	for _, pod := range pods {
 		pm[pod.Metadata.Name] = pod
 	}
-	pending := pendingJobs(pjs)
+	c.updatePendingJobs(pjs)
 	var syncErrs []error
 	if err := c.terminateDupes(pjs); err != nil {
 		syncErrs = append(syncErrs, err)
 	}
+
+	pjCh := make(chan kube.ProwJob, len(pjs))
```

Comment:
Not clear why we need this channel. Isn't it simpler to run the second loop you run directly?